beangulp/importers/csv.py

```diff
@@ -325,16 +334,22 @@ def get(row, ftype):
         if not is_ascending:
             entries = list(reversed(entries))
 
+        balances = {}
         # Add a balance entry if possible
-        if Col.BALANCE in iconfig and entries:
-            entry = entries[-1]
+        for entry in entries[::-1]:
```

Comment:
I think the `if Col.BALANCE in iconfig` part should stay, to avoid iterating the entries if it is known that there are no balances.

Reply:
That make sense. I will change it back to include the test.